MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -137,10 +137,10 @@ protected virtual string GetOutputFileExtension(StreamState state)
         /// </summary>
         private string GetOutputFilePath(StreamState state, EncodingOptions encodingOptions, string outputFileExtension)
         {
-            var data = GetCommandLineArguments("dummy\\dummy", encodingOptions, state, false);
-
-            data += "-" + (state.Request.DeviceId ?? string.Empty)
-                 + "-" + (state.Request.PlaySessionId ?? string.Empty);
+            var data = $"{state.MediaPath}-" +
+                       $"{state.UserAgent}" +
+                       $"{state.Request.DeviceId ?? string.Empty}-" +
+                       $"{state.Request.PlaySessionId ?? string.Empty}";
```

Comment:
Why this combination of string interpolation and concatenation?

Reply:
Ask Rider